geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/ConfigureEvictionThroughGfsh.java

```diff
@@ -0,0 +1,228 @@
+    assertThat(execution.getOutputText()).contains("Region \"/region2\" created on \"server\"");
+    assertThat(execution.getOutputText()).contains("Region \"/region3\" created on \"server\"");
+    assertThat(execution.getOutputText()).contains("Region \"/region4\" created on \"server\"");
+    assertThat(execution.getOutputText()).contains("Region \"/region5\" created on \"server\"");
+
+    execution = GfshScript
+        .of("connect --locator=localhost[10334]",
+            "create region --name=region6 --eviction-action=local-destroy --eviction-max-memory=1000 --type=REPLICATE_PERSISTENT")
+        .expectFailure().execute(gfsh);
+    assertThat(execution.getOutputText()).contains(
+        "ERROR: An Eviction Controller with local destroy eviction action is incompatible with");
+
+    execution = GfshScript
+        .of("connect --locator=localhost[10334]", "describe region --name=region1").execute(gfsh);
+    assertThat(execution.getOutputText()).containsPattern("region1")
+        .containsPattern("eviction-action\\s+| local-destroy")
+        .containsPattern("eviction-max-memory\\s+ | 1000");
+
+    execution = GfshScript
+        .of("connect --locator=localhost[10334]", "describe region --name=region2").execute(gfsh);
+    assertThat(execution.getOutputText()).containsPattern("region2")
+        .containsPattern("eviction-action\\s+| overflow-to-disk")
+        .containsPattern("eviction-max-memory\\s+ | 1000");
+
+    execution = GfshScript
+        .of("connect --locator=localhost[10334]", "describe region --name=region3").execute(gfsh);
+    assertThat(execution.getOutputText()).containsPattern("region3")
+        .containsPattern("eviction-action\\s+| overflow-to-disk")
+        .containsPattern("eviction-max-memory\\s+ | 1000");
+
+    execution = GfshScript
+        .of("connect --locator=localhost[10334]", "describe region --name=region4").execute(gfsh);
+    assertThat(execution.getOutputText()).containsPattern("region4")
+        .containsPattern("eviction-action\\s+| local-destroy")
+        .containsPattern("eviction-max-memory\\s+ | 1000");
+
+    execution = GfshScript
+        .of("connect --locator=localhost[10334]", "describe region --name=region5").execute(gfsh);
+    assertThat(execution.getOutputText()).containsPattern("region5")
+        .containsPattern("eviction-action\\s+| overflow-to-disk")
+        .containsPattern("eviction-max-memory\\s+ | 1000");
+
+
+  }
+
+  @Before
+  public void setup() throws IOException {
+    jarToDeploy = new File(gfsh.getTemporaryFolder().getRoot(), "ourJar.jar");
+
+    String classContents =
+        "import org.apache.geode.cache.util.ObjectSizer; import org.apache.geode.cache.Declarable;public class MySizer implements ObjectSizer, Declarable { public int sizeof(Object o) { return 10; } }";
+
+    JarBuilder jarBuilder = new JarBuilder();
+    jarBuilder.buildJar(jarToDeploy, classContents);
+  }
```

Comment:
I'm not sure it's necessary to put this in a {{@Before}} as it's only used in the one test that follows. Just declare it as a private method and call it in the test.